client/src/components/FormsUI/Textfield/index.tsx

```diff
@@ -0,0 +1,22 @@
+import { TextField } from '@material-ui/core';
+import { useField } from 'formik';
+
+const TextfieldWrapper = ({ name, ...otherProps }: any): JSX.Element => {
```

Comment:
For example, you can do something like:
```
type TextFieldWrapperProps = {
  fieldName : string,
} && TextFieldProps
```
I modified the prop `name` in order to avoid name collision - `name` is already a [native HTML attribute](https://html.spec.whatwg.org/multipage/indices.html#attributes-3:attr-form-name) (and as a result part of the [props](https://v4.mui.com/api/text-field/#props) TextField accepts too).